cross/transmission/Makefile

```diff
@@ -5,7 +5,7 @@ PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = https://github.com/transmission/transmission/releases/download/$(PKG_VERS)
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-DEPENDS = cross/curl cross/libevent
+DEPENDS = cross/curl cross/libevent cross/python38
```

Comment:
whenever python is required, use at least python310

Reply:
noted, but I don't think the issue has anything to do with python. going to try a different approach based on this consistent error across almost all the builds:
```
gmake[6]: *** No rule to make target 'third-party/utp/lib/libutp.a', needed by 'daemon/transmission-daemon'.  Stop.
```

Reply:
rolled back to original dependencies.